Aliucord/src/main/java/com/aliucord/api/PatcherAPI.java

```diff
@@ -76,6 +114,9 @@ public void run() {
         return _unpatch;
     }
 
+    /**
+     * Unpatches all patches.
```

Comment:
Removes all patches would probably sound better ngl

Reply:
Indeed, it sounds better